### DISCUSSION TITLE
val loss

### DISCUSSION THREAD

**Discussion by 1953678487** [open-mmlab/mmdetection#9601] [General]
I can only evaluate the mAP after each epoch is completed, and there is only training loss data in the log. How can I get the loss curve about the verification set?
